src/status_im/ui2/screens/chat/actions.cljs

```diff
@@ -60,14 +60,16 @@
                                                        :button-text (i18n/label :t/delete-chat)
                                                        :on-press    #(hide-sheet-and-dispatch [:chat.ui/remove-chat chat-id])}))}]))
 
-(defn leave-group-action [{:keys [chat-id] :as item}]
+(defn leave-group-action [item chat-id]
   (hide-sheet-and-dispatch [:bottom-sheet/show-sheet
                             {:content (fn []
                                         (common/alert {:title       (i18n/label :t/leave-group?)
                                                        :description (i18n/label :t/leave-chat-confirmation)
                                                        :context     item
                                                        :button-text (i18n/label :t/leave-group)
-                                                       :on-press    #(hide-sheet-and-dispatch [:chat.ui/leave-chat chat-id])}))}]))
+                                                       :on-press    #(do
+                                                                       (rf/dispatch [:navigate-back])
+                                                                       (hide-sheet-and-dispatch [:group-chats.ui/leave-chat-confirmed chat-id]))}))}]))
```

Comment:
Do we need to navigate back and hide the sheet?

Reply:
Yes! (exit the group details screen when leaving the group)